symbols/arm7.yml

```diff
@@ -68,6 +68,63 @@ arm7:
         This main function for the ARM7 subsystem. Contains the main event loop.
         
         No params.
+    - name: HardwareInterrupt
+      address:
+        EU-WRAM: 0x37FB488
```

Comment:
Can we add the normal (initial load) EU and NA addresses for these too?

Reply:
At first I didn't add them because calculating and verifying the offsets is quite tedious, but I guess I can just do a simple subtraction. It should be correct.

Reply:
Can we add JP addresses too? arm7 addresses should be identical across all three versions.

Reply:
Derived them with `symbols_vfill.py` and they are the same. Added them in.